opel-agent/values.yaml

```diff
@@ -0,0 +1,85 @@
+opentelemetry-collector:
+  mode: daemonset
+
+  fullnameOverride: otel-agent
+
+  extraEnvs:
+  - name: CORALOGIX_PRIVATE_KEY
+    valueFrom:
+      secretKeyRef:
+        name: coralogix-otel-privatekey
+        key: PRIVATE_KEY
+  - name: APP_NAME
+    value: production # Can be any static value
```

Comment:
@oded-dd Apparently a dynamic value is not working as expected, its taking the namespace of the agent itself instead of the pods. 
Also I deleted the subsystem because its going to be deprecated and its taking the servicename by default

Reply:
I think it can be resolved with the `attributesprocessor`

```
  resource:
    attributes:
    - key: 
      action: 
      value: 
```

Reply:
I will try again, but at the moment it doesnt work, so lets close it for now, and I'll keep trying

Reply:
I tried a lot, it didnt work, and I looked at the code itself and it seems not supported
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/coralogixexporter/client.go#L74